1.train_models/log_reg_plates_3_3p_5_cp_norm_data/nbconverted/plate_3_3p_5_train_log_reg_scv.py

```diff
@@ -294,14 +300,22 @@ def process_plates(_df):
 
         # Cumulative accuracy for all folds
         preds = logreg.predict(X_val)
+        preds_shuf = logreg.predict(X_val_shuf)
```

Comment:
This is a minor question, but I see that you trained your model using the original split rather than shuffled data. Initially, my assumption was that you would typically train another model on a shuffled feature space with the original labels, and then utilize the shuffled model on the original split (non shuffled) to evaluate its performance. 

Not saying this is an incorrect approach, I just want to learn c: !

Reply:
Ya, good question. Somebody asked me the same question the other day. This was my response:

There might be a difference between approaches, but I think it would be subtle. In contrast to just shuffling the data splits, the traditional approach would determine if meaningful patterns could be learned beyond what would be expected by chance. Whereas the other approach would determine if the patterns that were already learned could be used to find patterns in a new random and noisy dataset. This approach would prevent us from training another model and from storing as much data

Reply:
Thank you!